tests/e2e/default-mobile/wdio.conf.js

```diff
@@ -7,7 +7,16 @@ chai.use(require('chai-exclude'));
 exports.config = Object.assign(wdioBaseConfig.config, {
   suites: {
     all: [
-      '**/old-navigation.wdio-spec.js',
+      // '**/old-navigation.wdio-spec.js', this is now failing and should be investigated
+
+      // '**/reports/*.wdio-spec.js'
+      '../default-mobile/browser-compatibility/*.wdio-spec.js',
```

Comment:
For those files that have all tests are skipped, better comment the file here instead of adding `skip`